CHANGELOG.md

```diff
@@ -8,9 +8,29 @@
 
 ### Changed
 
+- Return a `Promise` from the below methods ([#948](https://github.com/Instabug/Instabug-React-Native/pull/948)):
+
+  - `Instabug.getTags`
+  - `Instabug.getUserAttribute`
+  - `Instabug.getAllUserAttributes`
+  - `Replies.hasChats`
+  - `Replies.getUnreadRepliesCount`
+  - `Surveys.getAvailableSurveys`
+  - `Surveys.hasRespondedToSurvey`
+
+  You should not pass it a callback and use the returned `Promise` as follows:
```

Comment:
suggestion: Do you think changing "methods" to "APIs" and "...and use..." to "but use..." would be a better wording here?